packages/qdrant-loader/tests/unit/connectors/git/test_git_connector.py

```
@@ -188,3 +188,156 @@ def get_metadata_for_file(file_path, content):
+
+class TestFetchById:
+    """Tests for the WS-1 fetch_by_id/list_entity_ids connector contract."""
+
+    @pytest.fixture
+    def mock_config(self):
+        """Fixture providing a mock Git repository configuration."""
+        return GitRepoConfig(
+            base_url=HttpUrl("https://github.com/test/repo.git"),
+            branch="main",
+            file_types=["*.md", "*.txt"],
+            token="test_token",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "token"

(S106)